spec/datadog/tracing/contrib/graphql/loading_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe 'loading graphql' do
```

Comment:
I would think that adding `require 'shellwords'` in this file is necessary so that it can be executed by itself? The only other place that requires this library is the top-level loading test that I can see.

Reply:
Thanks for the suggestion, I've added it !